cardano-cli/src/Cardano/CLI/Types/Common.hs

```diff
@@ -663,3 +667,12 @@ data PotentiallyCheckedAnchor anchorType anchor
   -- ^ Whether to check the hash or not (CheckHash for checking or TrustHash for not checking)
   }
   deriving (Eq, Show)
+
+-- | Type used for serialization when printing the hash of a transaction
+-- after having submitted it.
+newtype TxIdSubmission = TxIdSubmission {txhash :: T.Text}
```

Comment:
1. The name should be `TxSubmissionResult` or something similar, it's a result of a submitting a transaction and not tx id.
2. Why not use `TxId`? It has a JSON instance already

Reply:
> Why not use TxId? It has a JSON instance already.

^^ this

Reply:
Good point, I didn't know about `TxId`'s JSON output :+1: 

Changed :ok_hand: